MobileWalletTests/TariLibWrapperTests.swift

```diff
@@ -200,8 +197,24 @@ class TariLibWrapperTests: XCTestCase {
         XCTAssertEqual(wallet.availableBalance, 0)
         XCTAssertGreaterThan(wallet.pendingIncomingBalance, 0)
         XCTAssertGreaterThan(wallet.pendingOutgoingBalance, 0)
-               
-        //TODO confirm send tx
-        //TODO assert wallet.availableBalance
+       
+        //MARK: Complete sent transaction to bob
+        do {
+            let pendingOutboundTransaction = try wallet.pendingOutboundTransactions.at(position: 0)
+            try wallet.testCompleteSend(pendingOutboundTransaction: pendingOutboundTransaction)
+            
+            let broadcastedCompletedTx = try wallet.completedTransactions.at(position: 1)
```

Comment:
This may not be the same transaction as above. you will need to compare the transaction_id's to ensure they are the same.